text/0000-export-to-oci.md

```diff
@@ -0,0 +1,332 @@
+    ```
+    The problem seems to be that when the layers are added into the image we have the *layer diffID* reference, but after it was exported to OCI we have the *layer digest* inside the `blobs` folder. Maybe we need to store some kind of mapping: `layer diffID -> layer digest` to solve it.
+- We divide the results in two sections:
+  - Pre/Post Lifecycle execution: As mentioned, [skopeo](https://github.com/containers/skopeo) tool was used here and most of the time spent goes into this category. Open to optimal implementations here, maybe, one option could be to use [imgutil](https://github.com/buildpacks/imgutil) instead of [skopeo](https://github.com/containers/skopeo) and take advantage of the optimization to avoid re-uploading base layers.
+  - Lifecycle execution: We can see there is an increment of +10%, but probably this can be improved during the formal implementation
+
+
+# Migration
+[migration]: #migration
+
+## For the scope of this RFC
+
+- No breaking changes were identified
+
+## For removing Daemon support in near future
+
+I propose the following high level strategy to accomplish the goal
+
+- Approve this RFC and add the export to OCI layout format feature in Lifecycle
+- Implement in Pack a new workflow when the `-daemon` flag is enabled based on what we did with the PoC. The workflow could be enabled by the user in **experimental mode**
+  - Make noise in the community so people can help us using it
+  - Collect feedback about it
+  - Correct issues until we feel confortable
+- Turn off the actual `-daemon` implementation and replace it with the `-oci` by default in Pack
+  - Maybe we can also add a new flag to **re-use the old implementation** in case the user needs it
+  - Keep doing noise in the community
+  - Collect feedback about it
+  - Correct issues until we feel confortable
+- Schedule the removal of the Daemon support in Lifecycle version X.Y.Z.
+- Remove the actual daemon support from the code base for Lifecycle version X.Y.Z
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+- We could increase the disk space if we do not manage the duplication of saving the layers on disk. The proposal suggests to use symbolic links to reference layers on disk and avoid duplication.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- What other designs have been considered?
+  - Doing nothing, just keep exporting only to Daemon or Registry
+- Why is this proposal the best? [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format is a standard from which other tools can create a [OCI Runtime Specification bundle](https://github.com/opencontainers/runtime-spec/blob/v1.0.0/bundle.md) exporting to this format enables Platforms to implement any feature in the top of this format
```

Comment:
Should we add some examples of what some of those use cases may be? For example, [the  containerd request](https://github.com/buildpacks/lifecycle/issues/829) comes to mind. Loading to containerd seems more achievable if we had an OCI exported format.